tests/kuttl/tests/dataplane-deploy-no-nodes-test/01-assert.yaml

```diff
@@ -118,6 +118,8 @@ metadata:
     name: edpm-compute-no-nodes
 spec:
   backoffLimit: 6
+  ansibleExtraVars:
```

Comment:
this should be just extraVars, as that's what the field is called on the OpenStackAnsibleEE CRD.